documentation/examples/cqrs/infra/src/main/resources/application.conf

```
@@ -1 +1,2 @@
-play.crypto.secret=abcabcabcabcabcabc
\ No newline at end of file
+play.crypto.secret=abcabcabcabcabcabc
+play.http.secret.key=11111111000000
```

I believe `play.crypto.secret` is not needed anymore?

```suggestion
play.http.secret.key=11111111000000
```

---

removed all of the references on `application,conf`